osquery/tables/cloud/ec2_instance_tags.cpp

```diff
@@ -15,6 +15,9 @@
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 #include <osquery/utils/aws/aws_util.h>
+#ifdef WIN32
+#undef GetMessage
```

Comment:
Hm... We already have this [logic here](https://github.com/osquery/osquery/blob/e6fe15eb49660725e65dba1549932ed96e0a8c6e/plugins/logger/aws_log_forwarder.h#L28-L30). Is this pretty consistent for `aws` libs? Perhaps we could move the logic check in `aws_log_forwarder.h` into `osquery/utils/aws/aws_utils.h` and remove this one so it's a bit more generically applied to AWS libs?

Reply:
You want me to create a new .h file and factor this out in there?